### DISCUSSION TITLE
Select Multiple Accounts For Deletion

### DISCUSSION THREAD

**Discussion by dnburgess** [JulianPrieber/littlelink-custom#830] [Feature requests]
If a site gets spammed with new accounts that need to be deleted, it would be nice to have a check mark column down the side that would allow us to select multiple accounts and then delete in bulk rather than having to delete one account at a time.

**Comment by psjafri** [comment]
Today, I just noticed that suddenly, I have a lot of users, and most of them are spam.
I looked at my email, and in the span of a day, there were thousands of email notifications regarding new users and failed email notifications to the new user's email address. Luckily I set my Linkstack as manual verification.

![Screenshot 2025-03-24 at 09 12 04@2x](https://github.com/user-attachments/assets/ab1bfa1f-ee2c-450a-a8f8-f5373444756b)

The features to delete multiple users or pending users will be great for cleaning up the spam account.
